stf/src/sgx.rs

```diff
@@ -1,21 +1,28 @@
-use sgx_tstd as std;
-use std::{collections::HashMap, prelude::v1::*, vec};
-
+use crate::{
+	AccountId, Getter, Index, PublicGetter, ShardIdentifier, StatePayload, TrustedCall,
+	TrustedCallSigned, TrustedGetter, SUBSRATEE_REGISTRY_MODULE, UNSHIELD,
+};
 use codec::{Decode, Encode};
 use derive_more::Display;
 use log_sgx::*;
 use sgx_externalities::SgxExternalitiesTypeTrait;
-use sgx_runtime::{Balance, BlockNumber as L1BlockNumer, Runtime};
-use sp_core::{crypto::AccountId32, Pair, H256 as Hash};
+use sgx_runtime::{BlockNumber as L1BlockNumer, Runtime};
+use sgx_tstd as std;
+use sp_core::H256 as Hash;
 use sp_io::{hashing::blake2_256, SgxExternalitiesTrait};
 use sp_runtime::MultiAddress;
+use std::{collections::HashMap, prelude::v1::*, vec};
 use substratee_worker_primitives::BlockNumber;
 use support::{ensure, metadata::StorageHasher, traits::UnfilteredDispatchable};
 
-use crate::{
-	AccountId, Getter, Index, PublicGetter, ShardIdentifier, StatePayload, TrustedCall,
-	TrustedCallSigned, TrustedGetter, SUBSRATEE_REGISTRY_MODULE, UNSHIELD,
-};
+#[cfg(feature = "test_accounts")]
```

Comment:
I believe we should call this feature simply `test`, it does slightly more than setting up test accounts. Also, if we expand the feature later, we can just leave the name.

Reply:
Yes, I think that makes sense 👍

Reply:
Ah ok, I see that also makes sense, but we could also feature-gate the tests in this file, or does anything speak against that? Although, the test in this crate do not need the test_setup. Hmm, maybe it does not matter that much.

Reply:
I changed it to `test` now - for consistency sake, and because we might add 'real' tests under that feature in the future